build-info-api/src/main/java/org/jfrog/build/api/Vcs.java

```diff
@@ -41,6 +42,26 @@ public String toString() {
                 ", url='" + url + '\'';
     }
 
+    @Override
+    public boolean equals(Object o) {
```

Comment:
Is this method still needed, following the changes in the Jenkins Plugin code base?